python/lsst/sims/utils/Site.py

```diff
@@ -224,6 +225,25 @@ def __init__(self,
             msg += "instantiate your Site with name='LSST'"
             warnings.warn(msg)
 
+    def __eq__(self, other):
```

Comment:
Got it.  I will run the linter.  I suspect they are both fine.  It just depends on what one thinks is more readable.